packages/node_modules/@ciscospark/redux-module-teams/src/index.js

```diff
@@ -0,0 +1,66 @@
+import {fromJS} from 'immutable';
+
+export const STORE_TEAMS = `user/STORE_TEAMS`;
+export const UPDATE_STATUS = `user/UPDATE_STATUS`;
+
+export const initialState = fromJS({
+  items: {},
+  status: {
+    isFetching: false,
+    hasFetched: false
+  }
+});
+
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+  case STORE_TEAMS:
+    return state.setIn([`status`, `hasFetched`], true)
+      .mergeDeepIn([`items`], action.payload.teams);
+  case UPDATE_STATUS:
+    return state.mergeDeepIn([`status`], action.payload.status);
+  default:
+    return state;
+  }
+}
+
+
+export function fetchTeams(sparkInstance) {
+  return (dispatch) => {
+    dispatch(updateStatus({isFetching: true}));
+    return sparkInstance.internal.team.list()
+      .then((teams) => {
```

Comment:
Might want to `.catch` here too and set an error state.

Reply:
Yup yup.